04/styles/tablet.css

```diff
@@ -0,0 +1,52 @@
+.container {
+    background: url(../images/bg-big.svg) no-repeat;
+    /* nie rozumiem, czemu gdy ustawię background-size: cover TYLKO w global.css, to nie działa w tablet.css */
+    background-size: cover;
```

Comment:
Bo w linii wyżej je resetujesz. Gdybyś ustawił osobno tło i powtórzenie to powinno zadziałać :)